index.bs

```diff
@@ -12839,6 +12900,62 @@ running the algorithm for an {{AudioNode}}, using an <a>input
 buffer</a> and the value(s) of the {{AudioParam}}(s) of this
 {{AudioNode}} as the input for this algorithm.
 
+
+<div class="addition proposed" id="c2567-5">
+    <span class="marker">Proposed addition
+        <a href="https://github.com/WebAudio/web-audio-api/issues/2567">Issue 2567</a>.
+    </span>
+    Device-related error reporting via AudioContext
+    <div class="amendment-buttons"></div>
+    <ins>
+
+<h3 id="error-handling-on-a-running-audio-context">
+    Handling an error from System Audio Resources on the {{AudioContext}}</h3>
```

Comment:
```suggestion
    Handling an error from System Audio Resources on the and {{AudioContext}} |audioContext|</h3>
```

Reply:
I am not sure what this suggestion means. Can you clarify?

Reply:
I am assuming `and` is typo, and `{{AudioContext}}|audioContext|` means that "audioContext, an instance of AudioContext interface".

Reply:
yes, sorry! oops sorry about the typo! it's to match styles of other specs. For example:

https://www.w3.org/TR/geolocation/#request-a-position

Reply:
Having a variable within the title looks a bit weird, so I rephrase the prose to add `|audioContext|` variable in there.